scripts/ssl-manager.js

```diff
@@ -504,6 +544,23 @@ function SSLManager(config) {
         return { result: 0 };
     };
 
+    me.initWebrootMethod = function initWebrootMethod(webroot) {
+        config.webroot = me.initBoolValue(webroot);
+
+        if (config.webroot) {
+            nodeManager.getEntryNodeIps();
```

Comment:
what about response handling and logging?

Reply:
implemented via exec function

Reply:
didn't find any changes